src/IDPDiscoveryController.js

```diff
@@ -40,6 +41,12 @@ function (Okta, PrimaryAuthController, PrimaryAuthModel, IDPDiscoveryForm, IDPDi
 
       this.addListeners();
 
+      // If social auth is configured, 'socialAuthPositionTop' will determine
+      // the order in which the social auth and primary auth are shown on the screen.
+      if (options.settings.get('hasConfiguredButtons')) {
+        this.add(CustomButtons, {prepend: options.settings.get('socialAuthPositionTop')});
```

Comment:
this will add all custom buttons the user configures. In our case, that means PIV and also social idps. Just want to confirm that is what we want for the IdpDiscovery login form

Reply:
@mauriciocastillosilva-okta, I don't see social auth buttons appearing like PIV on the regular login page...is there something else I need to configure other than having `SOCIAL_AUTH` enabled as well as configuring a social idp?

Reply:
In additions, if we really going to this, the `constructor` looks quite similar to `PrimiaryAuthController.constructor` except how it creates `this.model`.

I wonder if you could refactor the parent construct to extra the difference into function, e.g `createModel`, and override in this child class.

Reply:
@haishengwu-okta can we do without the refactor for this PR? I made the changes to get it to work, but it caused ~25 test failures

Reply:
@mauriciocastillosilva-okta yeah that's the plan https://oktainc.atlassian.net/browse/OKTA-184467